flang/lib/Optimizer/Transforms/DebugTypeGenerator.cpp

```diff
@@ -286,7 +287,25 @@ mlir::LLVM::DITypeAttr DebugTypeGenerator::convertCharacterType(
     sizeInBits =
         charTy.getLen() * kindMapping.getCharacterBitsize(charTy.getFKind());
   } else {
-    return genPlaceholderType(context);
+    // In some situations, the len of the character is not part of the type but
```

Comment:
Should we mention "assumed length strings" instead of "some situations" ?

Reply:
Thanks for your review. I have updated the comment.